tests/RequestBuilderTest.php

```diff
@@ -9,37 +9,151 @@
 
 namespace Mailgun\Tests;
 
+use Http\Message\MultipartStream\MultipartStreamBuilder;
+use Http\Message\RequestFactory;
 use Mailgun\RequestBuilder;
+use PHPUnit\Framework\TestCase;
```

Comment:
Yes, but you will fail the "prefer-lowest" tests because there is no patch version specified. 

https://travis-ci.org/mailgun/mailgun-php/builds/234785059?utm_source=github_status&utm_medium=notification